forc-plugins/forc-node/src/run_opts.rs

```diff
@@ -0,0 +1,155 @@
+use std::{fmt::Display, path::PathBuf};
+
+/// Possible parameters to set while integrating with `fuel-core run`.
+#[derive(Debug, Default)]
+pub struct RunOpts {
```

Comment:
Did you consider using the [run::Command](https://github.com/FuelLabs/fuel-core/blob/856d5da980efce99a1b63cdd8e3f08bb95e42d60/bin/fuel-core/src/cli/run.rs#L102) directly from fuel-core and call [exec](https://github.com/FuelLabs/fuel-core/blob/856d5da980efce99a1b63cdd8e3f08bb95e42d60/bin/fuel-core/src/cli/run.rs#L654) to avoid calling the fuel-core CLI? 

I could imagine there being confusion if the user has the latest version of forc-node but an older version of fuel-core and expecting forc-node to use the latest version. It could also be confusing the other way around though. Just curious to hear your thoughts on this.

With the current approach, how would we know if a new version of fuel-core came out that was incompatible with the CLI args passed below (say, an arg name changed)?